test/e2e/node_reuse.go

```diff
@@ -537,7 +537,10 @@ func deleteTaint(taints []corev1.Taint, taintsToDelete []corev1.Taint) ([]corev1
 	return newTaints, deleted
 }
 
-func createNewM3machineTemplate(ctx context.Context, namespace string, newM3machineTemplateName string, m3machineTemplateName string, clusterClient client.Client, imageURL string, imageChecksum string, checksumType string, imageFormat string) {
+func createNewM3machineTemplate(ctx context.Context, namespace string, newM3machineTemplateName string, m3machineTemplateName string, clusterClient client.Client, imageURL string, imageChecksum string) {
```

Comment:
Since we're modifying this function anyway, let's change the name to a proper camelCase: `createNewM3MachineTemplate`.

Reply:
If function name, then variable names too? :)